Update Lisk Service setup, config, management, index and references - Closes #678 (#679)
LiskHQ/lisk-docs: modules/ROOT/pages/setup/index.adoc

The most convenient way to set up Lisk Service.

The xref:{url_setup_docker}[docker] image will install all required dependencies that are needed to run Lisk Service on your machine.

== Source code

> TIP: Install from source code if you want to customize the codebase of Lisk Service, for example by adjusting the API to be fully compatible your blockchain application.

Author: ```suggestion
TIP: Install from source code if you want to customize the codebase of Lisk Service, e.g. if you want to adjust the API to be fully compatible your blockchain application.
```
